src/bindings/python/fluxacct/accounting/test/test_job_archive_interface.py

```diff
@@ -338,10 +338,26 @@ def test_15_recalculate_usage_after_half_life_period(self):
 
         self.assertEqual(usage_factor, 2199.5)
 
+    # calling update_job_usage() in the same half-life period should NOT
+    # update usage factors for users
+    @mock.patch("time.time", mock.MagicMock(return_value=(10604799)))
```

Comment:
It might make more sense to set the `return_value` in an similar way to the above and below tests (e.g., `return_value=(10000000 + (604800 * 0.99))`)

Reply:
Agreed. I pushed up a `--fixup` commit that expands the `return_value` to match how `@mock.patch` is used amongst the other tests.